components/Jumbo.js

```diff
@@ -1,33 +1,54 @@
 import Button from './Button'
 
+const MIN_OPACITY = 0.1
+const OPACITY_START = 0.4
+
 export default ({top}) => {
-  const opacity = 1 - top / 800
+  const height =
+    typeof document === 'undefined'
```

Comment:
when is document ever undefined?

Reply:
next.js runs both on the server and client - so you don't have a guaranteed document/window/etc. In our case, we use `export`, which will render everything with node when building, which is the specific case this is covering.

Reply:
Thanks.